include/ginkgo/core/matrix/sellp.hpp

```diff
@@ -235,8 +235,8 @@ class Sellp : public EnableLinOp<Sellp<ValueType, IndexType>>,
     /**
      * @copydoc Sellp::val_at(size_type, size_type, size_type)
      */
-    value_type val_at(size_type row, size_type slice_set, size_type idx) const
-        noexcept
+    value_type val_at(size_type row, size_type slice_set,
+                      size_type idx) const noexcept
```

Comment:
formatting?

Reply:
Yes, clang-format did this. Should I leave it or change it back?